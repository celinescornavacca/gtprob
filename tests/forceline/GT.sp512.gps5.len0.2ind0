((((((((((HO2,(HO3,HO4)),(HO0,HO1)),((IO2,(IO3,IO4)),(IO0,IO1))),(((SL0,(SL1,((SL3,SL4),SL2))),((((TL3,TL4),TL2),TL1),TL0)),(((OL2,(OL3,OL4)),OL1),OL0))),(((((IC3,IC4),IC2),(IC0,IC1)),((((((DQ3,DQ4),DQ2),(DQ0,DQ1)),((EQ3,EQ4),((EQ1,EQ2),EQ0))),(((VD3,VD4),(VD1,VD2)),VD0)),((((((OO1,OO2),(OO3,OO4)),OO0),((PO1,(PO2,(PO3,PO4))),PO0)),(((DH3,DH4),DH0),(DH1,DH2))),(QF0,((QF2,(QF3,QF4)),QF1))))),(((NB3,NB4),(NB0,(NB1,NB2))),(((JH3,JH4),(JH0,(JH1,JH2))),((KH1,((KH3,KH4),KH2)),KH0))))),(((EA1,((EA3,EA4),EA2)),EA0),((((((MH1,MH2),(MH0,(MH3,MH4))),(((OT0,OT1),((OT3,OT4),OT2)),((PT0,(PT3,PT4)),(PT1,PT2)))),(((((NH3,NH4),NH0),(NH1,NH2)),(((OH1,OH2),OH0),(OH3,OH4))),((((JJ3,JJ4),JJ2),(JJ0,JJ1)),((KJ1,(KJ2,(KJ3,KJ4))),KJ0)))),((ZC0,(ZC1,((ZC3,ZC4),ZC2))),(((((NN3,NN4),NN0),(NN1,NN2)),((ON2,(ON3,ON4)),(ON0,ON1))),((JK1,JK2),((JK3,JK4),JK0))))),(((((CI1,(CI2,(CI3,CI4))),CI0),((DI1,(DI2,(DI3,DI4))),DI0)),((IH0,(IH3,IH4)),(IH1,IH2))),(((DG0,(DG3,DG4)),(DG1,DG2)),((EG2,(EG3,EG4)),(EG0,EG1))))))),((((((UC0,((UC2,(UC3,UC4)),UC1)),(((NI3,NI4),((NI1,NI2),NI0)),((((((RP3,RP4),RP2),RP1),RP0),(((SP3,SP4),SP2),(SP0,SP1))),(((XO3,XO4),XO0),(XO1,XO2))))),(((OB1,OB2),OB0),(OB3,OB4))),((((((((KO0,KO1),((KO3,KO4),KO2)),(((LO2,(LO3,LO4)),LO1),LO0)),(((YQ2,(YQ3,YQ4)),(YQ0,YQ1)),((ZQ0,ZQ1),(ZQ2,(ZQ3,ZQ4))))),((((((FM2,(FM3,FM4)),FM1),FM0),((NS0,(NS1,(NS2,(NS3,NS4)))),(((CT3,CT4),(CT0,(CT1,CT2))),((DT2,(DT3,DT4)),(DT0,DT1))))),((AL3,AL4),(AL0,(AL1,AL2)))),(((AF0,AF1),(AF2,(AF3,AF4))),((XJ0,(XJ1,(XJ2,(XJ3,XJ4)))),(((TQ2,(TQ3,TQ4)),(TQ0,TQ1)),(((UQ2,(UQ3,UQ4)),UQ1),UQ0)))))),(((HJ1,(HJ2,(HJ3,HJ4))),HJ0),((IJ1,IJ2),((IJ3,IJ4),IJ0)))),(((YL2,(YL3,YL4)),(YL0,YL1)),((ZL3,ZL4),(ZL0,(ZL1,ZL2))))),(((((LJ3,LJ4),LJ0),(LJ1,LJ2)),(MJ0,(((MJ3,MJ4),MJ2),MJ1))),((((OP1,OP2),OP0),(OP3,OP4)),(((TR3,TR4),((TR1,TR2),TR0)),(UR0,((UR2,(UR3,UR4)),UR1))))))),((((((TA3,TA4),TA2),(TA0,TA1)),(((((((BS0,BS1),((BS3,BS4),BS2)),((CS1,CS2),((CS3,CS4),CS0))),(((TH2,(TH3,TH4)),TH1),TH0)),((((WC3,WC4),WC2),WC1),WC0)),(((DK0,DK1),((DK3,DK4),DK2)),(EK0,(((EK3,EK4),EK2),EK1)))),((((XF0,XF1),(XF2,(XF3,XF4))),((YF1,YF2),((YF3,YF4),YF0))),((HE1,(HE2,(HE3,HE4))),HE0)))),((((((HL3,HL4),(HL1,HL2)),HL0),(((IL3,IL4),IL0),(IL1,IL2))),((KB3,KB4),((KB1,KB2),KB0))),((VA2,(VA3,VA4)),(VA0,VA1)))),(((AO0,((AO1,AO2),(AO3,AO4))),(((DO0,DO1),((DO3,DO4),DO2)),((EO3,EO4),(EO0,(EO1,EO2))))),((XR0,(XR1,(XR2,(XR3,XR4)))),((YR3,YR4),((YR1,YR2),YR0)))))),((((((PK3,PK4),PK2),(PK0,PK1)),((((JQ3,JQ4),(JQ1,JQ2)),JQ0),((KQ1,KQ2),((KQ3,KQ4),KQ0)))),(((VB0,(VB1,VB2)),(VB3,VB4)),((WB3,WB4),((WB1,WB2),WB0)))),(((((VC1,(VC2,(VC3,VC4))),VC0),((EJ0,((EJ2,(EJ3,EJ4)),EJ1)),((FJ3,FJ4),((FJ1,FJ2),FJ0)))),(((AB1,AB2),AB0),(AB3,AB4))),((((((((XP3,XP4),(XP1,XP2)),XP0),(((YP3,YP4),YP2),(YP0,YP1))),(((GI3,GI4),(GI1,GI2)),GI0)),((AG0,(((AG3,AG4),AG2),AG1)),((((US1,(US2,(US3,US4))),US0),((VS3,VS4),(VS0,(VS1,VS2)))),((SI1,SI2),((SI3,SI4),SI0))))),((((AK0,AK1),(AK2,(AK3,AK4))),(((BK1,BK2),(BK3,BK4)),BK0)),(((NR0,(((NR3,NR4),NR2),NR1)),((OR1,OR2),(OR0,(OR3,OR4)))),(((RM3,RM4),RM2),(RM0,RM1))))),(((PB0,((PB3,PB4),(PB1,PB2))),((QB0,(QB3,QB4)),(QB1,QB2))),(GB0,((GB2,(GB3,GB4)),GB1)))))))),((((((HF1,((HF3,HF4),HF2)),HF0),(((QK0,(QK1,((QK3,QK4),QK2))),((FN0,((FN1,FN2),(FN3,FN4))),(((GN1,GN2),GN0),(GN3,GN4)))),((LG1,LG2),((LG3,LG4),LG0)))),((((((((SG3,SG4),SG2),(SG0,SG1)),((TG0,TG1),(TG2,(TG3,TG4)))),(((YI0,(YI1,YI2)),(YI3,YI4)),((ZI0,ZI1),(ZI2,(ZI3,ZI4))))),(((NM3,NM4),(NM0,(NM1,NM2))),(((OM1,OM2),OM0),(OM3,OM4)))),(JA0,((JA3,JA4),(JA1,JA2)))),(((((((((HG1,((HG3,HG4),HG2)),HG0),(((IG3,IG4),IG2),(IG0,IG1))),(YE0,(((YE3,YE4),YE2),YE1))),((SD0,(SD3,SD4)),(SD1,SD2))),(((XB0,XB1),((XB3,XB4),XB2)),(((((KK3,KK4),KK0),(KK1,KK2)),((LK1,((LK3,LK4),LK2)),LK0)),((CR0,(((CR3,CR4),CR2),CR1)),(DR0,((DR3,DR4),(DR1,DR2))))))),((CD0,((CD1,CD2),(CD3,CD4))),(((HI3,HI4),((HI1,HI2),HI0)),(((II2,(II3,II4)),II1),II0)))),((((GM1,(GM2,(GM3,GM4))),GM0),(HM0,((HM2,(HM3,HM4)),HM1))),(((FD0,(FD1,(FD2,(FD3,FD4)))),((IK0,((IK2,(IK3,IK4)),IK1)),(((VQ0,VQ1),((VQ3,VQ4),VQ2)),(WQ0,((WQ1,WQ2),(WQ3,WQ4)))))),(((CB1,CB2),CB0),(CB3,CB4))))),(((((PG3,PG4),PG0),(PG1,PG2)),((QG0,QG1),((QG3,QG4),QG2))),(HB0,((HB1,HB2),(HB3,HB4))))))),((((((((YD0,(YD3,YD4)),(YD1,YD2)),((ZD0,((ZD2,(ZD3,ZD4)),ZD1)),((AE0,(AE3,AE4)),(AE1,AE2)))),((((VM0,((VM3,VM4),(VM1,VM2))),(((WM2,(WM3,WM4)),WM1),WM0)),(((EH0,(EH1,EH2)),(EH3,EH4)),(((((QT1,QT2),(QT3,QT4)),QT0),(RT0,(((RT3,RT4),RT2),RT1))),((XQ1,XQ2),((XQ3,XQ4),XQ0))))),(((OC1,(OC2,(OC3,OC4))),OC0),((PC1,(PC2,(PC3,PC4))),PC0)))),((IA0,IA1),(IA2,(IA3,IA4)))),((((((RK3,RK4),RK0),(RK1,RK2)),(SK0,(((SK3,SK4),SK2),SK1))),(((SA3,SA4),(SA1,SA2)),SA0)),(((YG0,((YG1,YG2),(YG3,YG4))),(((KP0,(KP1,KP2)),(KP3,KP4)),((LP1,LP2),((LP3,LP4),LP0)))),(((IE1,IE2),((IE3,IE4),IE0)),((ZF0,(((ZF3,ZF4),ZF2),ZF1)),(((IM0,IM1),(IM2,(IM3,IM4))),((JM2,(JM3,JM4)),(JM0,JM1)))))))),((((PD2,(PD3,PD4)),PD1),PD0),((QD0,QD1),((QD3,QD4),QD2)))),((((((TE1,TE2),(TE3,TE4)),TE0),((((((XL1,XL2),(XL3,XL4)),XL0),(((QS3,QS4),((QS1,QS2),QS0)),((RS1,RS2),((RS3,RS4),RS0)))),((WH1,((WH3,WH4),WH2)),WH0)),((((JN3,JN4),JN2),(JN0,JN1)),((KN2,(KN3,KN4)),(KN0,KN1))))),(((MD0,MD1),((MD3,MD4),MD2)),(((((VR1,VR2),(VR3,VR4)),VR0),(((WR2,(WR3,WR4)),WR1),WR0)),(((IR3,IR4),(IR1,IR2)),IR0)))),(((WG1,((WG3,WG4),WG2)),WG0),((XG3,XG4),(XG0,(XG1,XG2))))))),((((((OA0,(((OA3,OA4),OA2),OA1)),((WD0,(WD1,((WD3,WD4),WD2))),((XD2,(XD3,XD4)),(XD0,XD1)))),(((((((MT2,(MT3,MT4)),MT1),MT0),(((NT3,NT4),NT2),(NT0,NT1))),(((BM0,(((BM3,BM4),BM2),BM1)),((CM0,CM1),((CM3,CM4),CM2))),((((DS3,DS4),(DS1,DS2)),DS0),(((ES3,ES4),(ES1,ES2)),ES0)))),(((((SO1,((SO3,SO4),SO2)),SO0),(TO0,(TO1,(TO2,(TO3,TO4))))),(((RR2,(RR3,RR4)),(RR0,RR1)),((SR2,(SR3,SR4)),(SR0,SR1)))),(((RI1,RI2),RI0),(RI3,RI4)))),(((((((FG3,FG4),(FG1,FG2)),FG0),((GG1,GG2),(GG0,(GG3,GG4)))),((DC2,(DC3,DC4)),(DC0,DC1))),((((ZB1,ZB2),ZB0),(ZB3,ZB4)),(((((NQ1,NQ2),(NQ3,NQ4)),NQ0),(((OQ3,OQ4),OQ2),(OQ0,OQ1))),(((RH0,(((RH3,RH4),RH2),RH1)),((SH1,SH2),((SH3,SH4),SH0))),((SF0,(((SF3,SF4),SF2),SF1)),((TF1,TF2),((TF3,TF4),TF0))))))),((((((HD3,HD4),HD2),HD1),HD0),((ID3,ID4),(ID0,(ID1,ID2)))),((TC3,TC4),((TC1,TC2),TC0)))))),(((((((RE3,RE4),RE0),(RE1,RE2)),(((SE1,SE2),(SE3,SE4)),SE0)),(((BE3,BE4),(BE1,BE2)),BE0)),(((((TJ3,TJ4),TJ2),(TJ0,TJ1)),(UJ0,((UJ1,UJ2),(UJ3,UJ4)))),(((((JO0,(JO1,((JO3,JO4),JO2))),(((OS0,OS1),((OS3,OS4),OS2)),((PS1,PS2),((PS3,PS4),PS0)))),((JI0,(JI3,JI4)),(JI1,JI2))),((LL0,(LL1,(LL2,(LL3,LL4)))),(ML0,((ML1,ML2),(ML3,ML4))))),(CE0,((CE1,CE2),(CE3,CE4)))))),(((WF0,((WF2,(WF3,WF4)),WF1)),((((PN1,PN2),(PN3,PN4)),PN0),(((QN3,QN4),QN0),(QN1,QN2)))),((XE0,((XE2,(XE3,XE4)),XE1)),(((AR0,AR1),((AR3,AR4),AR2)),(BR0,(((BR3,BR4),BR2),BR1))))))),((((((((PR2,(PR3,PR4)),(PR0,PR1)),(((QR1,QR2),QR0),(QR3,QR4))),((((((LQ1,(LQ2,(LQ3,LQ4))),LQ0),(MQ0,((MQ1,MQ2),(MQ3,MQ4)))),(((GL1,GL2),(GL3,GL4)),GL0)),(((VF2,(VF3,VF4)),VF1),VF0)),((((GH1,GH2),(GH3,GH4)),GH0),(HH0,((HH3,HH4),(HH1,HH2)))))),(YB0,((YB1,YB2),(YB3,YB4)))),((((((BP3,BP4),BP2),BP1),BP0),((((ET3,ET4),(ET1,ET2)),ET0),(FT0,(FT1,(FT2,(FT3,FT4)))))),((XA0,XA1),(XA2,(XA3,XA4))))),(((((((VL0,(VL1,VL2)),(VL3,VL4)),(((WL2,(WL3,WL4)),WL1),WL0)),((((GC2,(GC3,GC4)),GC1),GC0),(((BG0,((BG1,BG2),(BG3,BG4))),((CG1,(CG2,(CG3,CG4))),CG0)),(JE0,(JE1,((JE3,JE4),JE2)))))),((((CJ3,CJ4),(CJ1,CJ2)),CJ0),((DJ0,DJ1),((DJ3,DJ4),DJ2)))),(((((((OD3,OD4),(OD1,OD2)),OD0),(((AM0,AM1),((AM3,AM4),AM2)),(((((TN3,TN4),TN2),TN1),TN0),((((ZP1,ZP2),(ZP3,ZP4)),ZP0),((AQ3,AQ4),((AQ1,AQ2),AQ0)))))),((HC1,(HC2,(HC3,HC4))),HC0)),(((UB3,UB4),UB0),(UB1,UB2))),(((CC0,(CC3,CC4)),(CC1,CC2)),(((((UG1,UG2),(UG3,UG4)),UG0),((((VG3,VG4),VG2),VG1),VG0)),(((AJ0,(((AJ3,AJ4),AJ2),AJ1)),(((BJ1,BJ2),(BJ3,BJ4)),BJ0)),((QE3,QE4),((QE1,QE2),QE0))))))),(((((PA0,(PA1,PA2)),(PA3,PA4)),(((QA1,QA2),QA0),(QA3,QA4))),((((((OI3,OI4),OI2),(OI0,OI1)),(PI0,((PI3,PI4),(PI1,PI2)))),(((FC3,FC4),(FC0,(FC1,FC2))),((AH0,((AH3,AH4),(AH1,AH2))),((BH0,(BH3,BH4)),(BH1,BH2))))),(((SS0,(SS1,(SS2,(SS3,SS4)))),((TS1,((TS3,TS4),TS2)),TS0)),(ZG0,(((ZG3,ZG4),ZG2),ZG1))))),(((((RA3,RA4),RA0),(RA1,RA2)),((((HS0,(HS3,HS4)),(HS1,HS2)),(IS0,(IS1,((IS3,IS4),IS2)))),((ME0,((ME2,(ME3,ME4)),ME1)),((((QO1,(QO2,(QO3,QO4))),QO0),(((RO3,RO4),(RO1,RO2)),RO0)),(UM0,((UM3,UM4),(UM1,UM2))))))),(((RD0,(RD1,RD2)),(RD3,RD4)),(((((LN3,LN4),(LN1,LN2)),LN0),(MN0,(MN1,(MN2,(MN3,MN4))))),((FH1,(FH2,(FH3,FH4))),FH0))))))),((((NA2,(NA3,NA4)),(NA0,NA1)),((((MO0,(MO1,(MO2,(MO3,MO4)))),(NO0,((NO3,NO4),(NO1,NO2)))),((((TD3,TD4),TD2),(TD0,TD1)),((UD1,(UD2,(UD3,UD4))),UD0))),((DB1,DB2),(DB0,(DB3,DB4))))),(((((((KG0,(KG1,KG2)),(KG3,KG4)),(((QL1,QL2),((QL3,QL4),QL0)),(((RL2,(RL3,RL4)),RL1),RL0))),((((UN0,UN1),((UN3,UN4),UN2)),(((VN3,VN4),(VN1,VN2)),VN0)),((((VI2,(VI3,VI4)),VI1),VI0),((WI1,WI2),(WI0,(WI3,WI4)))))),(((JG1,((JG3,JG4),JG2)),JG0),(((MG0,(MG3,MG4)),(MG1,MG2)),(((NG3,NG4),NG0),(NG1,NG2))))),(((BB2,(BB3,BB4)),(BB0,BB1)),(((ZE0,ZE1),(ZE2,(ZE3,ZE4))),(((((SM1,SM2),(SM3,SM4)),SM0),((TM3,TM4),((TM1,TM2),TM0))),((RG0,(RG3,RG4)),(RG1,RG2)))))),(((((((DL0,(DL3,DL4)),(DL1,DL2)),(((EL1,EL2),(EL3,EL4)),EL0)),(CF0,((CF3,CF4),(CF1,CF2)))),(((((((GT3,GT4),GT2),(GT0,GT1)),(((HT1,HT2),(HT3,HT4)),HT0)),((FL1,((FL3,FL4),FL2)),FL0)),((GJ1,(GJ2,(GJ3,GJ4))),GJ0)),((KF0,(KF1,KF2)),(KF3,KF4)))),((((JS3,JS4),(JS1,JS2)),JS0),((KS0,KS1),(KS2,(KS3,KS4))))),(((LA3,LA4),LA0),(LA1,LA2))))))),(((((DE0,((DE1,DE2),(DE3,DE4))),(((((AT1,AT2),(AT0,(AT3,AT4))),((BT0,(BT1,BT2)),(BT3,BT4))),(((CN3,CN4),CN0),(CN1,CN2))),((((JL2,(JL3,JL4)),JL1),JL0),(KL0,((KL1,KL2),(KL3,KL4)))))),((((ZM1,ZM2),(ZM3,ZM4)),ZM0),(((AN1,AN2),AN0),(AN3,AN4)))),(((HA0,(HA3,HA4)),(HA1,HA2)),((((((((MP0,MP1),((MP3,MP4),MP2)),(NP0,((NP3,NP4),(NP1,NP2)))),(GO0,((GO2,(GO3,GO4)),GO1))),(((XM3,XM4),(XM0,(XM1,XM2))),((YM2,(YM3,YM4)),(YM0,YM1)))),(((DF0,DF1),(DF2,(DF3,DF4))),((((YJ0,(YJ3,YJ4)),(YJ1,YJ2)),((((WK1,WK2),(WK3,WK4)),WK0),(((XK3,XK4),XK0),(XK1,XK2)))),((NK0,(NK1,((NK3,NK4),NK2))),(OK0,(OK1,(OK2,(OK3,OK4)))))))),((((RF0,(RF1,RF2)),(RF3,RF4)),(((FK1,((FK3,FK4),FK2)),FK0),(((FP2,(FP3,FP4)),(FP0,FP1)),(GP0,(GP1,(GP2,(GP3,GP4))))))),((((((LS2,(LS3,LS4)),LS1),LS0),((MS1,(MS2,(MS3,MS4))),MS0)),((ND1,ND2),((ND3,ND4),ND0))),((((FF1,((FF3,FF4),FF2)),FF0),(((GF1,GF2),GF0),(GF3,GF4))),((WE0,WE1),(WE2,(WE3,WE4))))))),((((BN0,(BN3,BN4)),(BN1,BN2)),((((LR3,LR4),LR2),(LR0,LR1)),(MR0,(MR1,(MR2,(MR3,MR4)))))),((XH1,XH2),((XH3,XH4),XH0)))))),((((((((BQ1,BQ2),(BQ0,(BQ3,BQ4))),(CQ0,((CQ2,(CQ3,CQ4)),CQ1))),((KM1,(KM2,(KM3,KM4))),KM0)),(((SC2,(SC3,SC4)),(SC0,SC1)),((((EI3,EI4),(EI1,EI2)),EI0),(((FI3,FI4),FI0),(FI1,FI2))))),(((IB3,IB4),IB2),(IB0,IB1))),((((((UE2,(UE3,UE4)),UE1),UE0),((((VE3,VE4),VE2),VE1),VE0)),(((RC1,RC2),(RC0,(RC3,RC4))),(((LI0,LI1),(LI2,(LI3,LI4))),(MI0,((MI3,MI4),(MI1,MI2)))))),((JC0,(JC1,(JC2,(JC3,JC4)))),((((KC3,KC4),KC2),KC1),KC0)))),((((SJ1,SJ2),(SJ3,SJ4)),SJ0),((ZO0,((ZO3,ZO4),(ZO1,ZO2))),(((AP2,(AP3,AP4)),AP1),AP0)))))))),((((AA0,(AA3,AA4)),(AA1,AA2)),((((((LB1,LB2),LB0),(LB3,LB4)),((MB1,MB2),((MB3,MB4),MB0))),((UH0,((UH1,UH2),(UH3,UH4))),((VH3,VH4),(VH0,(VH1,VH2))))),((((VJ0,VJ1),((VJ3,VJ4),VJ2)),(WJ0,((WJ3,WJ4),(WJ1,WJ2)))),(((JD3,JD4),(JD1,JD2)),JD0)))),((((((IF0,((IF2,(IF3,IF4)),IF1)),(((JF3,JF4),(JF1,JF2)),JF0)),(((WN1,(WN2,(WN3,WN4))),WN0),((XN0,(XN3,XN4)),(XN1,XN2)))),((((HQ0,(HQ1,((HQ3,HQ4),HQ2))),((IQ1,(IQ2,(IQ3,IQ4))),IQ0)),(HP0,(HP1,(HP2,(HP3,HP4))))),(UA0,(UA1,(UA2,(UA3,UA4)))))),(((RB0,(((RB3,RB4),RB2),RB1)),((((QC0,QC1),((QC3,QC4),QC2)),((VP0,((VP2,(VP3,VP4)),VP1)),(((WP3,WP4),WP2),(WP0,WP1)))),((((BC0,BC1),(BC2,(BC3,BC4))),(((YH1,YH2),((YH3,YH4),YH0)),(ZH0,((ZH3,ZH4),(ZH1,ZH2))))),((((((YO1,YO2),((YO3,YO4),YO0)),((YS0,(YS1,((YS3,YS4),YS2))),((ZS0,(ZS3,ZS4)),(ZS1,ZS2)))),((((GR0,(GR1,GR2)),(GR3,GR4)),((HR0,(HR1,HR2)),(HR3,HR4))),((KI1,KI2),((KI3,KI4),KI0)))),(ED0,(ED1,(ED2,(ED3,ED4))))),((FE0,(FE1,((FE3,FE4),FE2))),(((GE2,(GE3,GE4)),GE1),GE0)))))),(((((AD1,AD2),(AD3,AD4)),AD0),((BD1,BD2),((BD3,BD4),BD0))),((((PQ0,PQ1),((PQ3,PQ4),PQ2)),((QQ3,QQ4),(QQ0,(QQ1,QQ2)))),((((PJ1,PJ2),(PJ3,PJ4)),PJ0),(((QJ3,QJ4),QJ0),(QJ1,QJ2))))))),(((BA3,BA4),(BA0,(BA1,BA2))),((((YA0,((YA2,(YA3,YA4)),YA1)),((((AC1,AC2),(AC3,AC4)),AC0),(((((BL0,(BL3,BL4)),(BL1,BL2)),((CL0,CL1),(CL2,(CL3,CL4)))),((OG3,OG4),(OG0,(OG1,OG2)))),((DD3,DD4),(DD0,(DD1,DD2)))))),((((EB1,(EB2,(EB3,EB4))),EB0),(FB0,((FB3,FB4),(FB1,FB2)))),(((((IT2,(IT3,IT4)),IT1),IT0),(JT0,(JT1,(JT2,(JT3,JT4))))),((((KE1,KE2),((KE3,KE4),KE0)),((LE1,LE2),(LE0,(LE3,LE4)))),((NC0,((NC2,(NC3,NC4)),NC1)),((((LM3,LM4),((LM1,LM2),LM0)),(MM0,(MM1,((MM3,MM4),MM2)))),(((YN2,(YN3,YN4)),(YN0,YN1)),((ZN1,ZN2),(ZN0,(ZN3,ZN4)))))))))),((DA0,DA1),(DA2,(DA3,DA4)))))))),(((((((((((((BO0,(BO1,(BO2,(BO3,BO4)))),(CO0,(CO1,((CO3,CO4),CO2)))),(((ZJ1,ZJ2),ZJ0),(ZJ3,ZJ4))),((((EF3,EF4),EF2),EF1),EF0)),((EE3,EE4),(EE0,(EE1,EE2)))),(((((AI1,AI2),AI0),(AI3,AI4)),(((BI3,BI4),BI2),(BI0,BI1))),(((NJ0,(NJ1,NJ2)),(NJ3,NJ4)),(((OJ3,OJ4),OJ2),(OJ0,OJ1))))),(((TP0,((TP3,TP4),(TP1,TP2))),((UP1,UP2),((UP3,UP4),UP0))),(((NL0,NL1),((NL3,NL4),NL2)),((ER0,(ER1,(ER2,(ER3,ER4)))),((FR1,FR2),(FR0,(FR3,FR4))))))),((((PH3,PH4),PH2),(PH0,PH1)),((QH3,QH4),((QH1,QH2),QH0)))),((LF0,(((LF3,LF4),LF2),LF1)),(((MF1,MF2),(MF3,MF4)),MF0))),(((KD1,(KD2,(KD3,KD4))),KD0),((LD3,LD4),(LD0,(LD1,LD2))))),((((((XC1,((XC3,XC4),XC2)),XC0),(YC0,(YC1,((YC3,YC4),YC2)))),(((RN0,((RN1,RN2),(RN3,RN4))),((SN3,SN4),(SN0,(SN1,SN2)))),(((DM0,((DM2,(DM3,DM4)),DM1)),((EM2,(EM3,EM4)),(EM0,EM1))),((VK1,(VK2,(VK3,VK4))),VK0)))),(((RJ2,(RJ3,RJ4)),(RJ0,RJ1)),((MK0,((MK2,(MK3,MK4)),MK1)),(((UK1,(UK2,(UK3,UK4))),UK0),(((PM3,PM4),((PM1,PM2),PM0)),((QM0,QM1),((QM3,QM4),QM2))))))),((((IP2,(IP3,IP4)),(IP0,IP1)),(JP0,((JP3,JP4),(JP1,JP2)))),(((((GK1,GK2),(GK3,GK4)),GK0),(((HK3,HK4),HK0),(HK1,HK2))),(((((NE2,(NE3,NE4)),NE1),NE0),((((DN3,DN4),DN0),(DN1,DN2)),(EN0,(EN1,(EN2,(EN3,EN4)))))),(((((FO3,FO4),(FO0,(FO1,FO2))),((ZR0,((ZR3,ZR4),(ZR1,ZR2))),((AS0,(AS3,AS4)),(AS1,AS2)))),(PL0,(((PL3,PL4),PL2),PL1))),((UF1,((UF3,UF4),UF2)),UF0))))))),(CA0,((CA2,(CA3,CA4)),CA1))),(((((FA1,FA2),((FA3,FA4),FA0)),((((((OF0,(OF3,OF4)),(OF1,OF2)),((PF0,(PF3,PF4)),(PF1,PF2))),((((RQ1,RQ2),((RQ3,RQ4),RQ0)),(SQ0,(SQ1,(SQ2,(SQ3,SQ4))))),((QI1,(QI2,(QI3,QI4))),QI0))),((WA0,WA1),((WA3,WA4),WA2))),(KA0,(KA1,((KA3,KA4),KA2))))),(((LC0,((LC3,LC4),(LC1,LC2))),((((MC3,MC4),MC2),MC1),MC0)),(((MA1,MA2),((MA3,MA4),MA0)),(((((LH1,LH2),LH0),(LH3,LH4)),((((PP3,PP4),PP2),(PP0,PP1)),((QP0,(QP3,QP4)),(QP1,QP2)))),((((((((KT2,(KT3,KT4)),(KT0,KT1)),((LT3,LT4),((LT1,LT2),LT0))),(((EP3,EP4),EP2),(EP0,EP1))),(CK0,((CK3,CK4),(CK1,CK2)))),((((XI3,XI4),XI2),XI1),XI0)),(((NF1,NF2),NF0),(NF3,NF4))),(((ZA2,(ZA3,ZA4)),(ZA0,ZA1)),(((((TI3,TI4),(TI0,(TI1,TI2))),((UI1,(UI2,(UI3,UI4))),UI0)),((BF1,BF2),((BF3,BF4),BF0))),(((GD2,(GD3,GD4)),GD1),GD0)))))))),((((((((FQ1,FQ2),FQ0),(FQ3,FQ4)),((GQ1,GQ2),(GQ0,(GQ3,GQ4)))),(((SB1,SB2),(SB0,(SB3,SB4))),(TB0,(TB1,(TB2,(TB3,TB4)))))),(((((EC1,EC2),EC0),(EC3,EC4)),(((YK2,(YK3,YK4)),(YK0,YK1)),((ZK1,(ZK2,(ZK3,ZK4))),ZK0))),(((JB1,JB2),(JB3,JB4)),JB0))),((((VO1,VO2),(VO3,VO4)),VO0),((WO2,(WO3,WO4)),(WO0,WO1)))),((((((((FS3,FS4),(FS1,FS2)),FS0),(GS0,((GS2,(GS3,GS4)),GS1))),(((JR0,((JR3,JR4),(JR1,JR2))),(((KR1,KR2),KR0),(KR3,KR4))),(UO0,(((UO3,UO4),UO2),UO1)))),(((WS0,((WS1,WS2),(WS3,WS4))),((XS0,XS1),((XS3,XS4),XS2))),((TK3,TK4),((TK1,TK2),TK0)))),((((CP0,CP1),((CP3,CP4),CP2)),((DP0,DP1),(DP2,(DP3,DP4)))),(CH0,((CH3,CH4),(CH1,CH2))))),((GA0,((GA3,GA4),(GA1,GA2))),(((((UL3,UL4),UL2),(UL0,UL1)),(((HN0,(HN1,HN2)),(HN3,HN4)),(((IN2,(IN3,IN4)),IN1),IN0))),(((OE0,(OE3,OE4)),(OE1,OE2)),(PE0,(((PE3,PE4),PE2),PE1))))))))));
