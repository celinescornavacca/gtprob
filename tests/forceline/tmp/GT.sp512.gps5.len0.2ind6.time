173312.22user 64.13system 48:08:28elapsed 100%CPU (0avgtext+0avgdata 9592096maxresident)k
72inputs+8outputs (0major+12982971minor)pagefaults 0swaps
