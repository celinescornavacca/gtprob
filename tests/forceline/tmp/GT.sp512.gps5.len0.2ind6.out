-5063.27762767
